spec/controllers/dashboards_controller_spec.rb

```diff
@@ -109,7 +109,7 @@ def load_table(klass, options)
         get :push
 
         expect(flash.alert).to eq('No preview version available')
-        expect(Version.production_version).to be_blank
+        expect(Version.current_production).to be_blank
```

Comment:
the spec is testing to make sure that push returns a flash error and that nothing is pushed to production. The last expectation in the example is kind of dumb, but I guess it doesn't hurt to have it.